app/build.gradle.kts

```diff
@@ -155,6 +156,7 @@ android {
       isDebuggable = false
       isMinifyEnabled = runProguard.toBoolean()
       isShrinkResources = runProguard.toBoolean()
+      signingConfig = if(properties.get("maestroTest") == "true") getByName("debug").signingConfig else null
```

Comment:
Instead of setting a `null` value for signing config, can we instead replace it with this?

```kt
if (maestroTest.toBoolean()) {
  signingConfig = getByName("debug").signingConfig
}
```

This allows us to use the proper signing key when making a release build in Bitrise.